src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -367,6 +367,9 @@
     <Compile Include="$(BclSourcesRoot)\System\Globalization\GlobalizationMode.Windows.cs" />
     <Compile Include="$(BclSourcesRoot)\System\Threading\ClrThreadPoolBoundHandle.Windows.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(FeatureAppX)' == 'true' or '$(EnableWinRT)' == 'true'">
```

Comment:
`EnableWinRT` is not ever defined in CoreCLR. This can be just `'$(FeatureAppX)' == 'true'`.